styx-scheduler-service/src/main/java/com/spotify/styx/api/SchedulerResource.java

```diff
@@ -148,4 +189,9 @@ public SchedulerResource(
     // todo: change payload to a struct returning the triggerId as well so the user can refer to it
     return Response.forPayload(workflowInstance);
   }
+
+  private static String arg(String name, RequestContext rc) {
```

Comment:
I think you can just inline this as there is only one place calling this method.